authentication/ssh_agent_signer.go

```diff
@@ -72,7 +73,10 @@ func NewSSHAgentSigner(input SSHAgentSignerInput) (*SSHAgentSigner, error) {
 		return nil, err
 	}
 	signer.key = matchingKey
-	signer.formattedKeyFingerprint = formatPublicKeyFingerprint(signer.key, true)
+        signer.formattedKeyFingerprint, err = formatPublicKeyFingerprint(signer.key, true)
```

Comment:
Indentation is weird here.

Reply:
@twhiteman, 

Question: This does not affect consumers of triton-go, correct? I.e. they don't need to change any code due to this change?

Answer:  Correct.

This functionality just expands the type of private key material that authentication.NewPrivateKeySigner() can use.

Is an example of how it is used:
https://github.com/joyent/triton-go/blob/master/examples/compute/instances/main.go#L73

Currently you can see here that we were only able to parse PKCS1 Private Keys.  Any other keys required the use of the ssh-agent.
https://github.com/joyent/triton-go/blob/master/authentication/private_key_signer.go#L51

sdc-cloudapi supports DSA,RSA, and ECDSA. (https://github.com/joyent/sdc-cloudapi/blob/master/lib/auth.js#L63) but I'm hesitant to add DSA due to its fragility. https://github.com/golang/go/issues/40337